packages/jupyter-chat/style/chat.css

```diff
@@ -106,3 +106,12 @@
   padding: 2px 0;
   font-weight: bold;
 }
+
+.jp-chat-open {
+  max-width: calc(100% - 140px);
+  overflow: hidden;
+}
+
+.lm-AccordionPanel-titleLabel {
```

Comment:
We should probably be more specific on the selector, to avoid targeting other DOM elements in the page.